.github/workflows/renovate-changeset.yml

```diff
@@ -0,0 +1,87 @@
+# Modelled on https://github.com/backstage/backstage/blob/5083c8024deffbdf454983900f02780d14b55a0b/.github/workflows/sync_renovate-changesets.yml#L6
+
+name: Sync Renovate changeset
+on:
+  pull_request_target:
+    paths:
+      - '.github/workflows/renovate-changeset.yml'
+      - 'pnpm-lock.yaml'
+
+jobs:
+  generate-changeset:
+    runs-on: ubuntu-latest
+    if: github.actor == 'renovate[bot]' && github.repository == 'seek-oss/skuba'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+          ref: ${{ github.head_ref }}
+          token: ${{ secrets.SEEK_OSS_CI_GITHUB_TOKEN }}
+      - name: Configure Git
+        run: |
+          git config --global user.email 34733141+seek-oss-ci@users.noreply.github.com
+          git config --global user.name 'Github changeset workflow'
+      - name: Generate changeset
+        uses: actions/github-script@v7
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
+        with:
+          script: |
```

Comment:
Do actions requite this to be inline?

Reply:
Not directly, but it's a bit gross either way https://github.com/actions/github-script?tab=readme-ov-file#run-a-separate-file